virttest/libvirt_xml/vm_xml.py

```diff
@@ -2713,11 +2725,75 @@ def __init__(self, virsh_instance=base.virsh):
                                parent_xpath='/',
                                tag_name='memorytune',
                                attribute='vcpus')
+        accessors.XMLElementList(property_name="nodes",
+                                 libvirtxml=self,
+                                 parent_xpath='/',
+                                 marshal_from=self.marshal_from_nodes,
+                                 marshal_to=self.marshal_to_nodes,
+                                 has_subclass=True)
+        accessors.XMLElementList(property_name="monitors",
+                                 libvirtxml=self,
+                                 parent_xpath='/',
+                                 marshal_from=self.marshal_from_monitors,
+                                 marshal_to=self.marshal_to_monitors,
+                                 has_subclass=True)
 
         super(MemoryTuneXML, self).__init__(
             virsh_instance=virsh_instance)
         self.xml = '<memorytune/>'
 
+    @staticmethod
+    def marshal_from_nodes(item, index, libvirtxml):
```

Comment:
what's the index usage?

Reply:
Then should we del it like other marshal functions definition?

Reply:
@kylazhang After running tests with deleting these vars and keeping them, I found no difference between these 2 solutions (The amount of left xml temp files are the same). Therefore deleting them explicitly seems not necessary.

Reply:
ok, thanks